kernel/kernel-api/src/test/scala/io/delta/kernel/internal/checkpoints/CheckpointerSuite.scala

```diff
@@ -224,7 +221,7 @@ class CheckpointerSuite extends AnyFunSuite
   }
 }
 
-object CheckpointerSuite {
+object CheckpointerSuite extends MockTableClientUtils with VectorTestUtils {
```

Comment:
why extend `MockTableClientUtils`? `VectorTestUtils` makes sense as it contains the `stringVector` etc.

Reply:
Fixed.